inspirehep/modules/theme/static/scss/format/detailed-record.scss

```diff
@@ -18,8 +18,66 @@
  * 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
  */
 
- $purple: #293E51;
- $darken: 8%;
+@import "../base/variables";
+@import "../../node_modules/bootstrap-sass/assets/stylesheets/_bootstrap";
+@import "../base/header";
+
+$purple: #293E51;
+$darken: 8%;
```

Comment:
Are these variables only useful here? If not, please move them to a more general file. BTW, not a fan of the `$purple` name, better use names like `$main-header-color` or something like that.